src/main/scala/com/amazon/deequ/analyzers/catalyst/DeequFunctions.scala

```diff
@@ -79,6 +80,12 @@ object DeequFunctions {
     val statefulDataType = new StatefulDataType()
     statefulDataType(column)
   }
+
+  def stateful_kll(column: Column, sketchSize: Int = KLLSketch.defaultSketchSize,
```

Comment:
Please format according to our [coding conventions](https://github.com/databricks/scala-style-guide)

Reply:
Updated.